CSharpMath/Atoms/Factories/MathAtoms.cs

```diff
@@ -322,7 +322,7 @@ public static class MathAtoms {
          { "threeunderdot", new Accent("\u20E8") } //not in iosMath
       });
     
-    public static LargeOperator Operator(string name, bool? limits, bool noLimits = false) => new LargeOperator(name, limits, noLimits);
+    public static LargeOperator Operator(string displayChar, bool? limits, bool noLimits = false, string name = "") => new LargeOperator(displayChar, limits, noLimits, name);

```

Comment:
LargeOperators can be multiple characters (e.g. `\lim`) so `displayChar` is not a good name.

Reply:
Then which name we can give ?

Reply:
`display`  / `displays` / `displayAs` / `displayChars` / `displayString`

Reply:
Ok